src/Numerics/DGMethods/DGModel_kernels.jl

```diff
@@ -302,7 +302,12 @@ Computational kernel: Evaluate the volume integrals on right-hand side of a
         @unroll for k in 1:Nqk
             ijk = i + Nq * ((j - 1) + Nq * (k - 1))
             @unroll for s in 1:num_state_conservative
-                tendency[ijk, s, e] = local_tendency[k, s]
+                if β != 0
+                    T = α * local_tendency[k, s] + β * tendency[ijk, s, e]
```

Comment:
We may want to prefetch tendency earlier, but you'd have to time it to see. Probably a minor performance thing anyway.

Reply:
That was my main worry.

Reply:
I would guess it is pretty minor. Probably in the noise of all the other kernels.

Reply:
If you did want to prefetch, you could add to the top
```julia
local_tendency[k, s] =  β != 0 ? tendency[ijk, s, e] : -zero(FT)
```
then add later (before the first use)
```julia
local_tendency[k, s] =  β * local_tendency[k, s]
```

Reply:
> If you did want to prefetch

Nevermind, that doesn't work...

Reply:
> Nevermind, that doesn't work...

Because then you'd be scaling the current tendency by `α * β` so I guess really you'd want
```julia
local_tendency[k, s] =  (β / α) * local_tendency[k, s]
```

Reply:
that should be fine, since I don't think anyone is going to run this with `α=0`. 

The other option would be to scale by `α` at every addition to `local_tendency[k, s]`.